releasenotes/notes/snmp-profiles-9711c7d2307e9362.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Copy SNMP profiles shipped by the integration during installation.
```

Comment:
I don't think this PR requires a mention in the changelog at all, since AFAIK it's only related to the integrations, so it's expected for a user who uses a version of an integration that includes SNMP profiles to see them in the Agent package that include that version of the integration

Reply:
Thanks, I changed it to `other`, is that enough?